cfme/common/__init__.py

```diff
@@ -365,6 +365,48 @@ def step(self, *args, **kwargs):
         self.prerequisite_view.toolbar.policy.item_select('Manage Policies')
 
 
+class ComparableMixin:
+    """
+    Mixin for comparing entities. Should be added to Collection class.
+    Args:
+            provider: An InfraProvider instance
+            entities_list: A list of entities to compare
+    Constants:
+            DROPDOWN_TEXT: (str) the text in the Configuration dropdown for comparing entities
+            NAV_STRING: (str) used as second arg in call to navigate_to()
+            COMPARE_VIEW: (view class) view class to create and return when filtering on provider
+            COMPARE_APP_VIEW: (view class) view class to create and return when no provider
```

Comment:
What do mean by `no provider`?

Reply:
Also I'm trying to think of a more descriptive name. Something like, `COMPARE_VIEW` and `COMPARE_VIEW_NO_PROVIDER`, what do you think?

Reply:
What about COMPARE_VIEW_PROVIDER and COMPARE_VIEW_ALL ? As you said, no provider is a bit misleading.